src/symbolize/symbolizer.rs

```diff
@@ -1695,7 +1695,31 @@ mod tests {
         use crate::test_helper::with_bpf_symbolization_target_addrs;
 
         with_bpf_symbolization_target_addrs(|handle_getpid, subprogram| {
-            println!("handle_getpid: {handle_getpid:#x}, subprogram: {subprogram:#x}");
+            let src = symbolize::Source::Kernel(symbolize::Kernel::default());
+            let symbolizer = Symbolizer::new();
+            let result = symbolizer
+                .symbolize(
+                    &src,
+                    symbolize::Input::AbsAddr(&[handle_getpid, subprogram]),
+                )
+                .unwrap();
+            let handle_getpid_sym = result[0].as_sym().unwrap();
+            assert_eq!(handle_getpid_sym.name, "handle__getpid");
+            // BPF code information is very spotty. For the very address
```

Comment:
see above, line info is (semantically) segments of instructions, not individidual instruction. So you should always find some line_info with binary search.